libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts

```diff
@@ -1,6 +1,6 @@
 import { NgModule } from '@angular/core';
```

Comment:
I think all uses of this module are now gone with your changes, so it can probably be removed.

Reply:
It is still used in one place in KirbyModule:
https://github.com/kirbydesign/designsystem/blob/56ffd1a4cddd05a61d2c533bc060a5d58a6ef3d2/libs/designsystem/src/lib/kirby.module.ts#L171

However, I could copy the contents of KirbyIonicModule to KirbyModule, and then delete KirbyIonicModule.

Reply:
Ahh alright, probably better to leave as is and not further clutter KirbyModule 👍